### PR TITLE
Added dashboard navbar for use on aepp-identity dashboard changes.

### PR DESCRIPTION
**Pull request by Philip-Nunoo**

### CODE REVIEW

**Comment by Philip-Nunoo** [comment]
I believe this is necessary in other to manage the style for the navbar.

**Comment by davidyuk** [comment]
Basically, it is AeHeader component in mobile view.

File: src/components/DashboardNavbar/DashboardNavbar.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.dashboard-navbar {
+  padding-top: 10px;
+  padding-bottom: 10px;
+
+  .title {
+    font-family: Roboto;
```
Comment (davidyuk) [inline src/components/DashboardNavbar/DashboardNavbar.scss]: Roboto font is set for the whole page by default, I don't think that it should be specified explicitly.

File: src/components/DashboardNavbar/DashboardNavbar.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    font-size: 24px;
+    font-weight: bold;
+    text-align: center;
+    color: #1e1e1e;
```
Comment (davidyuk) [inline src/components/DashboardNavbar/DashboardNavbar.scss]: This color is defined in `variables.scss` as `$anthracite`.

File: src/components/DashboardNavbar/DashboardNavbar.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  }
+
+  .left{
+    float:left;
```
Comment (davidyuk) [inline src/components/DashboardNavbar/DashboardNavbar.scss]: Please, add a space after the colon.

File: src/components/DashboardNavbar/DashboardNavbar.vue [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<template>
+  <div class="dashboard-navbar">
+    <div class="left"><slot name="left" /></div>
+    <div class="right"><slot name="right" /></div>
+    <div class="title"><slot /></div>
+  </div>
+</template>
+<script src='./DashboardNavbar.js'/>
```
Comment (davidyuk) [inline src/components/DashboardNavbar/DashboardNavbar.vue]: And again a mix of " and '.

**Pull request closed without merging**